File: Python/DesignHashset.py
## Leetcode Problem - 705

class MyHashSet:
    def __init__(self) -> None:
        self.set = []

    def add(self,key):
        if key not in self.set:
            self.set.append(key)

    def remove(self,key):
        if key in self.set:
            self.set.delete(key)

    def contains(self,key):
        if key in self.set:
            return True
        else:
            return False

File: Javascript/DesignHashset.js
var MyHashSet = function(){
    this.set = []
}

MyHashSet.prototype.add = function(key){
    if (!this.set.includes(key)){
        this.set.push(key)
    }
}

MyHashSet.prototype.remove = function(key){
    if (this.set.includes(key)){
        this.set.splice(this.set.indexOf(key),1)
    }
}

MyHashSet.prototype.contains = function(key){
    if (this.set.includes(key)){
        return true
    }else{
        return false
    }
}